### DISCUSSION TITLE
Sending certain emails to BCC instead of CC when replying-all ?

### DISCUSSION THREAD

**Discussion by jb55** [neomutt/neomutt#4222] [Development]
When responding to github comments, it has a special `reply+AAALE...@reply.github.com` email which contains secret data that allows people to post on that thread looking like its me. Sometimes I like to add this to BCC when I CC other mailing lists so that I don't leak this token. Is there a way to automatically send certain emails like reply.github.com to bcc when replying-all ?

**Comment by flatcap** [comment]
Sorry this took a while...

It's an interesting idea, but I can't think of a way to do it in NeoMutt.
But...

You could fudge it, if you:

- Have NeoMutt config: `set edit_headers = yes`
- You can can script your editor.

This Vim config works: (you may need to adjust the path for your temp files)

```vim
autocmd BufReadPost /tmp/mutt/neomutt-* g/reply+\i\+@reply.github.com/s/^To:/Bcc:/e   
```

On reading the file, it does an automatic `s/To/Bcc/` if it sees `@reply.github.com`

Hope this helps

**Reply by jb55** [reply]
awesome, will try that. thanks!

**Comment by jb55** [comment]
On Wed, Apr 17, 2024 at 04:19:50AM GMT, Richard Russon wrote:
>Sorry this took a while...
>
>It's an interesting idea, but I can't think of a way to do it in NeoMutt.
>But...
>
>You could fudge it, if you:
>
>- Have NeoMutt config: `set edit_headers = yes`
>- You can can script your editor.
>
>This Vim config works: (you may need to adjust the path for your temp files)
>
>```vim
>autocmd BufReadPost /tmp/mutt/neomutt-* ***@***.***/s/^To:/Bcc:/e
>```
>
>On reading the file, it does an automatic `s/To/Bcc/` if it sees ***@***.***`
>
>Hope this helps

One thing I did recently was when generating patches locally for ingest
into notmuch-insert: I construct the patch from github, writing Bcc: with
the github reply address to the mail locally.

This enables me to do github code review from my inbox, in theory.. The
hope was neomutt would pick up the Bcc on reply-all, but it seems to
completely ignore it. not sure if its this a bug.
